.github/workflows/main.yml

```diff
@@ -209,6 +212,7 @@ jobs:
   fablab-smoketest-teardown:
     name: Teardown SmokeTest
     if: always()
```

Comment:
The timeout is shorter than the sleep, so the sleep will always trigger the timeout

Reply:
Good catch.